src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -23,7 +23,7 @@
 import swervelib.parser.SwerveParser;
 
 public class Drive extends SubsystemBase {
-  SwerveDrive drive;
+  private SwerveDrive drive;
```

Comment:
`private final` I think

Reply:
Because of the weird try/catch clause, this isn't super viable.

Reply:
gotcha, resolving